tests/test_crud.py

```diff
@@ -112,5 +112,4 @@ def run_playbook(module, extra_vars=None, limit=None):
 @pytest.mark.parametrize('module', MODULES)
 def test_crud(tmpdir, module, record):
     run = run_playbook_vcr(tmpdir, module, record=record)
-    print(run.stdout.read())
```

Comment:
this is not needed because because run playbook already will print stuff?

Reply:
Yes, i noticed the output (in case of a failure) was doubled.